connect/basic-auth-extension/src/main/java/org/apache/kafka/connect/rest/basic/auth/extenstion/PropertyFileLoginModule.java

```diff
@@ -51,17 +55,17 @@
     private static Map<String, Properties> credentialPropertiesMap = new ConcurrentHashMap<>();
 
     @Override
-    public void initialize(Subject subject, CallbackHandler callbackHandler,
-                           Map<String, ?> sharedState, Map<String, ?> options) {
+    public void initialize(Subject subject, CallbackHandler callbackHandler, Map<String, ?> sharedState, Map<String, ?> options) {
         this.callbackHandler = callbackHandler;
         fileName = (String) options.get(FILE_OPTIONS);
-        if (fileName != null && !fileName.isEmpty()) {
+        if (fileName != null && !fileName.isEmpty() && !credentialPropertiesMap.containsKey(fileName)) {
```

Comment:
Should we log a warning in an `else` block for this `if` block? If somebody does not specify the filename, this login module will always fail authentication, right?